game_options.cpp

```diff
@@ -110,12 +131,27 @@ void test_game_options()
   }
   // Two random game options differ in their key-action maps especially
   {
+    const int rng_seed = 111;

```

Comment:
Nice, adapting tests to facilitate new behavior! Note that this is what mediors do :innocent: